lua/nvim-web-devicons.lua

```diff
@@ -1482,6 +1488,7 @@ local filetypes = {
   ["go"] = "go",
   ["godot"] = "godot",
   ["graphql"] = "graphql",
+  ["gql"] = "gql",
```

Comment:
A map of the filetype to the existing icon should be enough `["gql"] = "graphql",`.

Please try that.

Reply:
That's exactly what I tried first but I couldn't get it to work

Reply:
Right you are; it's necessary when fetching via extension, apologies.